tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/array-test.bal

```diff
@@ -153,3 +153,23 @@ function testGetFromFrozenArray() returns int {
 
     return -1;
 }
+
+type Age object {
+    public int age;
+    public function __init(int age) {
+    	 self.age = age;
+    }
+};
+
+function testObjectDynamicArrayFilling() {
```

Comment:
Is this test complete/called?

Reply:
Fixed https://github.com/ballerina-platform/ballerina-lang/pull/20749/commits/6469ab1a9367249f3dae2e65642ec135bcc29c26#diff-73b31fb7a29b74dcbbff2be6c403b505R216